### PR TITLE
Doc 657 add undocumented env vars

### PR DESCRIPTION
**Pull request by ayatnkw**

### CODE REVIEW

File: docs/hosting/configuration/environment-variables/security.md [hunk 1]
```diff
@@ -16,3 +16,7 @@ hide:
 | Variable | Type  | Default  | Description |
 | :------- | :---- | :------- | :---------- |
 | `N8N_BLOCK_ENV_ACCESS_IN_NODE` | Boolean | `false` | Whether to allow users to access environment variables in expressions and the Code node (false) or not (true). |
+| `N8N_RESTRICT_FILE_ACCESS_TO` | String |  | Limits access to files in these directories. Provide multiple files as a colon-separated list ("`:`"). |
+| `N8N_BLOCK_FILE_ACCESS_TO_N8N_FILES` | Boolean | `true` | Set to `true` to block access to all files in the ".n8n" directory and user defined configuration files. |
```
Comment (StarfallProjects) [inline docs/hosting/configuration/environment-variables/security.md:20]:
```suggestion
| `N8N_BLOCK_FILE_ACCESS_TO_N8N_FILES` | Boolean | `true` | Set to `true` to block access to all files in the `.n8n` directory and user defined configuration files. |
```

File: docs/hosting/configuration/environment-variables/deployment.md [hunk 2]
```diff
@@ -19,8 +19,9 @@ This page lists the deployment configuration options for your self-hosted n8n in
 | :------- | :---- | :------- | :---------- |
 | `N8N_EDITOR_BASE_URL` | String | - | Public URL where users can access the editor. Also used for emails sent from n8n. |
 | `N8N_CONFIG_FILES` | String | - | Use to provide the path to any JSON [configuration file](/hosting/configuration/configuration-methods/). |
-| `N8N_DISABLE_UI` | Boolean | `false` | Disable the UI (true) or not (false). |
-| `N8N_TEMPLATES_ENABLED` | Boolean | `true` | Enable workflow templates (true) or disable (false). |
+| `N8N_DISABLE_UI` | Boolean | `false` | Set to `true` to disable the UI. |
+| `N8N_PREVIEW_MODE` | Boolean | `false` | Set to `true` to runs in preview mode. |
```
Comment (StarfallProjects) [inline docs/hosting/configuration/environment-variables/deployment.md:23]:
```suggestion
| `N8N_PREVIEW_MODE` | Boolean | `false` | Set to `true` to run in preview mode. |
```

Comment (StarfallProjects) [reply docs/hosting/configuration/environment-variables/deployment.md:23]: What is preview mode?

File: docs/hosting/configuration/environment-variables/deployment.md [hunk 3]
```diff
@@ -34,15 +35,17 @@ This page lists the deployment configuration options for your self-hosted n8n in
...
- |
+| `VUE_APP_URL_BASE_API` | String | `http://localhost:5678/` | Used when building the `n8n-editor-ui` package manually to set how the frontend can reach the backend API. Refer to [Configure the Base URL](/hosting/configuration/configuration-examples/base-url/). |
 | `N8N_HIRING_BANNER_ENABLED` | Boolean | `true` | Whether to show the n8n hiring banner in the console (true) or not (false). |
 | `N8N_PUBLIC_API_SWAGGERUI_DISABLED` | Boolean | `false` | Whether the Swagger UI (API playground) is disabled (true) or not (false). |
 | `N8N_PUBLIC_API_DISABLED` | Boolean | `false` | Whether to disable the public API (true) or not (false). |
 | `N8N_PUBLIC_API_ENDPOINT` | String | `api` | Path for the public API endpoints. |
 | `N8N_GRACEFUL_SHUTDOWN_TIMEOUT` | Number | `30` | How long should the n8n process wait (in seconds) for components to shut down before exiting the process. |
+| `N8N_DEV_RELOAD` | Boolean | `false` | Set to `true` to automatically reload or restart when changes occur in the source code. |
```
Comment (StarfallProjects) [inline docs/hosting/configuration/environment-variables/deployment.md:49]: Suggest making it explicit that this is for working on the source code.

File: docs/hosting/configuration/environment-variables/external-hooks.md [hunk 4]
```diff
@@ -18,3 +18,4 @@ You can define external hooks that n8n executes whenever a specific operation ru
 | Variable | Type  | Description |
 | :------- | :---- | :---------- |
 | `EXTERNAL_HOOK_FILES` | String | Files containing external hooks. Provide multiple files as a colon-separated list ("`:`"). |
+| `EXTERNAL_FRONTEND_HOOKS_URLS` | String | URLs to external frontend hooks files. Provide multiple files as a colon-separated list ("`:`"). |
```
Comment (StarfallProjects) [inline docs/hosting/configuration/environment-variables/external-hooks.md:21]: Might want to rephrase this. I'm guessing the files aren't external - it's the hooks that are external?

File: docs/hosting/configuration/environment-variables/logs.md [hunk 5]
```diff
@@ -28,6 +28,7 @@ This page lists environment variables to set up logging for debugging. Refer to
 | `DB_LOGGING_ENABLED` | Boolean | `false` | Whether to enable database-specific logging. |
 | `DB_LOGGING_OPTIONS` | Enum string: `query`, `error`, `schema`, `warn`, `info`, `log`  | `error` | Database log output level. To enable all logging, specify `all`. Refer to [TypeORM logging options](https://orkhan.gitbook.io/typeorm/docs/logging#logging-options){:target=_blank .external-link} |
 | `DB_LOGGING_MAX_EXECUTION_TIME` | Number | `1000` | Maximum execution time (in milliseconds) before n8n logs a warning. Set to `0` to disable long running query warning. |
+| `CODE_ENABLE_STDOUT` | Boolean | `false` | Set to`true` to send Code node logs to process's stdout. |
```
Comment (StarfallProjects) [inline docs/hosting/configuration/environment-variables/logs.md:31]:
```suggestion
| `CODE_ENABLE_STDOUT` | Boolean | `false` | Set to `true` to send Code node logs to process's stdout. |
```

Comment (StarfallProjects) [reply docs/hosting/configuration/environment-variables/logs.md:31]: Can we explain the circumstances when this is useful?

Comment (ayatnkw) [reply docs/hosting/configuration/environment-variables/logs.md:31]: So something like "Set to`true` to send Code node logs to process's stdout for debugging, monitoring, or logging purposes. Recommended only in non-production environments for security reasons." 

Should I include the second sentence as we disabled this by default for a reason stated [here](https://community.n8n.io/t/n8n-ignores-n8n-log-level-env-var-value/10916/15)? What do you think?

File: docs/hosting/configuration/environment-variables/external-hooks.md (hunk 4 above)
Comment (ayatnkw) [reply docs/hosting/configuration/environment-variables/external-hooks.md]: Made some changes in the [latest commit](https://github.com/n8n-io/n8n-docs/pull/2147/commits/a04501518f7eeaad3f26f48ccf0aa61ee2ac8c11)

File: docs/hosting/configuration/environment-variables/deployment.md (hunk 3 above)
Comment (ayatnkw) [reply docs/hosting/configuration/environment-variables/deployment.md]: Made changes [here](https://github.com/n8n-io/n8n-docs/pull/2147/commits/a04501518f7eeaad3f26f48ccf0aa61ee2ac8c11)

File: docs/hosting/configuration/environment-variables/logs.md (hunk 5 above)
Comment (StarfallProjects) [reply docs/hosting/configuration/environment-variables/logs.md]: That sounds good

**Review by Joffcom — APPROVED** [review]
They look good to me, Thanks for doing this @ayatnkw

**Pull request merged**